design/ceph/object/store.md

```diff
@@ -73,7 +74,83 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
   preservePoolsOnDelete: true
 ```
 
-If there is a `zone` section in object-store configuration, then the pool section in the ceph-object-zone resource will be used to define the pools.
+#### Pools shared by multiple CephObjectStore
+
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. And for this option to work, `preservePoolsOnDelete` must be set to 'true'. Now the object stores can consume same pool isolated with different namespaces. User can create via [Pool CRD](/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md), this is need to present before the object store is created.
+
+```yaml
+  radosNamespaces:
+    metadataPoolName: rgw-meta-pool
+    dataPoolName: rgw-data-pool
+  preservePoolsOnDelete: true
+```
+
+To create the pools that will be shared by multiple object stores, create the following CephBlockPool CRs:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-meta-pool
+  spec:
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+    failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+The `.rgw.root` pool must be defined before the object store is created. This pool is used for storing the root info for all object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-root
+  spec:
+    name: .rgw.root
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+```
+
+The pools for this configuration will be created as below:
+
+```bash
+# ceph osd pool ls
+.rgw.root
+rgw-meta-pool.rgw.control:my-store
+rgw-meta-pool.rgw.meta:my-store
+rgw-meta-pool.rgw.log:my-store
+rgw-meta-pool.rgw.otp:my-store
+rgw-data-pool.rgw.buckets.data:my-store
+rgw-data-pool.rgw.buckets.index:my-store
+```
+
+#### For mulisite use case
+
+If there is a `zone` section in object-store configuration, then the pool creation will performed by the [CephObjectZone CRD](/Documentation/CRDs/Object-Storage/ceph-object-zone-crd.md). The `CephObjectStore` CRD will include below section to specify the zone name.
```

Comment:
Did you mean the namespace creation?
```suggestion
If there is a `zone` section in object-store configuration, then the RADOS namespace creation will performed by the [CephObjectZone CRD](/Documentation/CRDs/Object-Storage/ceph-object-zone-crd.md). The `CephObjectStore` CRD will include below section to specify the zone name.
```

Reply:
Sorry I don't understand this part. This is the normal case, if zone is specified in the CephObjectStore CRD then pool creation(without radosnamespaces) will handle by CephObjectZone CRD